sc-platform-api/src/api/article/article.controller.ts

```diff
@@ -0,0 +1,31 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import { ArticleService } from './article.service';
+import { ApiEndpoint, GetUser } from '../../documentation/decorators';
+import { ArticleResponse, CreateArticleDto } from '@student-council-platform/utils';
+import { AdminGuard } from '../../security/roles/admin.guard';
+import { DbArticle } from '../../database/entities/DbArticle';
+import { ArticleMapper } from './article.mapper';
+import { AccessGuard } from '../../security/jwt/access/access.guard';
+import { ArticleDocumentation } from '../../documentation/article';
+
+@Controller('articles')
+export class ArticleController {
+  constructor (
+    private readonly articleService: ArticleService,
+    private readonly articleMapper: ArticleMapper,
+  ) {}
+
+  @ApiEndpoint({
+    summary: 'Create a new article',
+    guards: [AccessGuard, AdminGuard],
+    documentation: ArticleDocumentation.CREATE,
+  })
+  @Post('/')
+  async create (
+    @Body() createArticleDto: CreateArticleDto,
```

Comment:
Here you do not receive the dto, but the body, so rename this parameter
```suggestion
    @Body() body: CreateArticleDto,
```